### DISCUSSION TITLE
tukaani-project Repository access

### DISCUSSION THREAD

**Discussion by amazon-aws** [community/community#116515] [Repositories]
### Select Topic Area

General

### Body

Hi Team,

I am working on milvus build & getting error while downloading this repo: https://github.com/tukaani-project/xz/archive/v5.4.4.tar.gz.
error: Failed to download from mirror set
     error: File does not have the expected hash:
     url: https://github.com/tukaani-project/xz/archive/v5.4.4.tar.gz

Last week, there was no issue coming. Can you please check

**Comment by GordonSmith** [comment]
This also affects vcpkg builds...

**Reply by amazon-aws** [reply]
Yes, this is affecting & due to this Milvus  bin & libs are not generating.

**Comment by Nueck** [comment]
I also encountered this problem, how to solve it

**Comment by qqii** [comment]
As of 2024-04-09, the [repository](https://tukaani.org/xz-backdoor/) is now back online. The maintainer has an update page [here](https://tukaani.org/xz-backdoor/).
